code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -470,8 +470,8 @@ datum/reagent/medicine/morphine
 
 datum/reagent/medicine/morphine/on_mob_life(var/mob/living/M as mob)
 	M.status_flags |= IGNORESLOWDOWN
-	if(current_cycle >= 36)
-		M.sleeping += 3
+	if(current_cycle >= 12)
```

Comment:
morphine was the replacement for sleep toxin actually

if/when I do pain and painkillers morphine will effect that too